packages/ai-commands/src/sql/chat.ts

```diff
@@ -58,7 +58,7 @@ export async function chatSql(
 
   try {
     const response = await openai.chat.completions.create({
-      model: 'gpt-3.5-turbo-0125',
+      model: 'gpt-4o-mini',
```

Comment:
We should use a specific version of `gpt-4o-mini-2024-07-18` from https://platform.openai.com/docs/models/gpt-4o-mini. This is to help with reproducibility.